index.js

```diff
@@ -443,7 +443,7 @@ ShareDbMongo.prototype.getOpsToSnapshot = function(collectionName, id, from, sna
   this._getOps(collectionName, id, from, to, options, function(err, ops) {
     if (err) return callback(err);
     var filtered = getLinkedOps(ops, null, snapshot._opLink);
-    var err = checkOpsFrom(collectionName, id, filtered, from);
+    var err = (options || {}).ignoreMissingOps ? null : checkOpsFrom(collectionName, id, filtered, from);
```

Comment:
Might be a bit more readable and future-proof if we just make sure options is initialised in its own line:

```suggestion
    options = options || {};
    var err = options.ignoreMissingOps ? null : checkOpsFrom(collectionName, id, filtered, from);
```

Reply:
I also personally find use of the ternary a bit weird here, but this is more of a nitpick, so 🤷🏼‍♂️ :

```suggestion
    var err = null;
    if (!options.ignoreMissingOps) {
      err = checkOpsFrom(collectionName, id, filtered, from);
    }
```